docs/schemas.md

```diff
@@ -22,8 +22,8 @@ These are the field types that are supported in the schemas for this plugin.
 | array                       | Field of type `array`                         | See [Arrays and maps](#arrays-and-maps) for details on this field.                                       |
 | map                         | Field of type `map`                           | See [Arrays and maps](#arrays-and-maps) for details on this field.                                       |
 | timestamp                   | Field of type `timespan`                      | Random timestamp value                                                                                   |                                                                               
-| stringTimestamp             | Field of type `string` but coding a Timestamp | Localdatetime.now formatted as string                                                                    |
-| longTimestamp               | Field of type `long` but coding a Timestamp   | Localdatetime.now formatted as long                                                                      |
+| stringTimestamp             | Field of type `string` but coding a Timestamp | `localdatetime.now` formatted as string                                                                    |
+| longTimestamp               | Field of type `long` but coding a Timestamp   | `localdatetime.now` formatted as long                                                                      |
```

Comment:
Should `LocalDateTime` be capitalized, as it's the name of a class, or are we using lowercase for consistency?

Reply:
That's true; I'm going to fix it right now :) Thank you!